vars/main.yml

```diff
@@ -1 +1,5 @@
 ---
```

Comment:
@alexander-medved https://github.com/verygood-ops/ansible-port-knocking/blob/master/defaults/main.yml

Reply:
Done

Reply:
@max-lobur Could you please merge my PR?